internal/tensorzero-node/src/database.rs

```diff
@@ -204,6 +202,39 @@ impl DatabaseClient {
         napi_call!(&self, get_datapoint, params, GetDatapointParams)
     }
 
+    #[napi]
+    pub async fn list_datapoints(&self, params: String) -> Result<String, napi::Error> {
+        // Deserialize ListDatapointsRequestWithDatasetName from the params string
+        let request_with_dataset: ListDatapointsRequestWithDatasetName =
+            serde_json::from_str(&params).map_err(|e| napi::Error::from_reason(e.to_string()))?;
+
+        // Convert to GetDatapointsParams
+        let get_params = GetDatapointsParams {
```

Comment:
Why don't we just expose this to the frontend directly and have the napi function be `list_datapoints(&self, dataset_name: String, params: String)`?